GetandTake/Pages/Category.cshtml.cs

```diff
@@ -0,0 +1,21 @@
+using GetandTake.Models;
+using GetandTake.Models.DTOs.ListDTO;
+using GetandTake.Services.Abstracts;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace GetandTake.Pages
+{
+    public class CategoryModel : PageModel
+    {
+        private readonly ICategoryService _categoryService;
+        public IEnumerable<Category> Categories { get; private set; }
```

Comment:
Let's add empty spaces between all of these members to avoid them overlapping on each other (same case for other page models)